tests/test_tablib.py

```diff
@@ -1180,6 +1182,18 @@ def test_xlsx_bad_dimensions(self):
             data = tablib.Dataset().load(fh, read_only=False)
         self.assertEqual(data.height, 3)
 
+    def test_xlsx_raise_ValueError_on_cell_write_during_export(self):
+        """Test that the process handles errors which might be raised
+        when calling cell setter."""
+        setter_mock = Mock(wraps=Cell.value.fset, side_effect=[ValueError, '1'])
+        mock_property = Cell.value.setter(setter_mock)
+        with patch.object(Cell, 'value', mock_property):
+            data.append((1,))
+            _xlsx = data.export('xlsx')
+            wb = load_workbook(filename=BytesIO(_xlsx))
+            # note the Cell is mocked so we don't get the 'real' value
+            self.assertEqual(None, wb.active['A1'].value)
```

Comment:
ok - have used an 'array' value which openpyxl cannot handle.  This is probably clearer but means we are also testing openpyxl code, instead of mocking as an external dependency.  So if they ever change their implementation to handle arrays, this test will break.

Reply:
Also prefer this simpler version, we'll just have to update the test if they change the implementation.

Reply:
Thanks for reviewing guys - happy to go with the simpler approach.  Over to you if you are happy to merge.